vault/external_plugin_test.go

```diff
@@ -935,23 +935,15 @@ func mountPlugin(t *testing.T, sys *SystemBackend, pluginName string, pluginType
 	}
 }
 
-func unmountPlugin(t *testing.T, sys *SystemBackend, pluginName string, pluginType consts.PluginType, version, path string) {
+func unmountPlugin(t *testing.T, sys *SystemBackend, pluginType consts.PluginType, path string) {
```

Comment:
Removed some unnecessary inputs. Note: This function doesn't actually fail the test if it doesn't unmount anything, but I refrained from fixing that directly in this PR to stop its size ballooning.